packages/http-server/src/request-handlers/create-exchange.ts

```diff
@@ -1,69 +1,70 @@
-import type { RequestHandler, OfferingsApi, ExchangesApi, SubmitRfqCallback } from '../types.js'
+import type { OfferingsApi, ExchangesApi, SubmitRfqCallback } from '../types.js'
 import { Rfq } from '@tbdex/protocol'
 import type { ErrorDetail } from '@tbdex/http-client'
 
 import { CallbackError } from '../callback-error.js'
+import { Request, Response } from 'express'
 
 type CreateExchangeOpts = {
-  callback: SubmitRfqCallback
+  callback?: SubmitRfqCallback
   offeringsApi: OfferingsApi
   exchangesApi: ExchangesApi
 }
 
-export function createExchange(options: CreateExchangeOpts): RequestHandler {
+export async function createExchange(req: Request, res: Response, options: CreateExchangeOpts): Promise<any> {
```

Comment:
Alternative approach... 

```suggestion
export async function createExchange(req: Request, res: Response, options: CreateExchangeOpts): Promise<void> {
```

And then rather than something like this,

```typescript
return res.status(400).json({ errors: [{ detail: 'replyTo must be a valid url' }] })
```

We would do this...

```typescript
res.status(400).json({ errors: [{ detail: 'replyTo must be a valid url' }] })
return 
```

Benefit here is the function signature is intentional in how we expect it to be used (but maybe I'm wrong, maybe we want calling functions to have access to the `Response` type which is currently being returned?)

Reply:
Oh shoot, disregard, this is intended to be used within express's dependencies

Reply:
I'm updating to `Promise<void>` since express.js doesn't look at any return type from this function. It seems to be a pure side effect. Express docs: https://expressjs.com/en/5x/api.html#app.get